packages/design-systems/src/components/composite/Datagrid/types.ts

```diff
@@ -19,8 +19,8 @@ type CommonDatagridProps<TData extends Record<string, unknown>> = {
 
   // Filter
   enableColumnFilters?: boolean;
-  systemFilter?: QueryFilter;
-  defaultFilter?: QueryFilter;
+  systemFilter?: Record<string, unknown>;
+  defaultFilter?: Record<string, unknown>;
```

Comment:
すいません、意図としてはsystemFilter/defaultFilterの構造を型で制限したいので `BuildableFilter` にしてもらいたかったです。

これだと、今以上にあり得ない形のデータもObjectとして渡せるようになってしまうので、さらによくない状態ですので。

```suggestion
  systemFilter?: BuildableFilter;
  defaultFilter?: BuildableFilter;
```